docs/en/deploy/Running-Alluxio-On-a-HA-Cluster.md

```diff
@@ -351,6 +351,9 @@ Below are the steps to add a new master to a live cluster:
 
 Note: Adding to an already shut down cluster still requires adding only single master at a time.
 
+Note: When adding a master to a single master cluster, you should shut down and update configuration for the existing master.
```

Comment:
Reference which properties should be configured/looked at? or even just another doc regarding configuring EJ